### PR TITLE
vim-patch:6f4754b9f725

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/6f4754b9f7253d7e4ba527064a24aff1acdb1e8f

(skipping most of the "cosmetic" changes that don't apply cleanly, especially in `messages.txt`. also skipped all error codes above 1000 as they don't appear in our source?)

### CODE REVIEW

**Review by seandewar — COMMENTED** [review]
could include https://github.com/vim/vim/commit/6f4754b9f7253d7e4ba527064a24aff1acdb1e8f#diff-7e9292cae1f2ba70dd5b17d2d162693a91044ada6ac99e9c3e8917f32878c097L893
vim moved that error to the docs for `'opendevice'`, but nvim doesn't have that error (actually `opendevice` is unsupported in nvim, but we still have its docs for some reason :thinking:)

File: runtime/doc/diff.txt [hunk 1]
```diff
@@ -324,8 +324,9 @@ After setting this variable, reload the syntax script: >
 
 FINDING THE DIFFERENCES					*diff-diffexpr*
 
-The 'diffexpr' option can be set to use something else than the standard
-"diff" program to compare two files and find the differences. *E959*
+The 'diffexpr' option can be set to use something else than the internal diff
+support or the standard "diff" program to compare two files and find the
+differences. *E959*
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/diff.txt:329]: it's gone now ;)

[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/doc/diff.txt:329]: the rest of the paragraph is fine tho

File: runtime/doc/nvim_terminal_emulator.txt [hunk 2]
```diff
@@ -167,7 +167,7 @@ gdb window	A terminal window in which "gdb vim" is executed.  Here you
 program window	A terminal window for the executed program.  When "run" is
 		used in gdb the program I/O will happen in this window, so
 		that it does not interfere with controlling gdb.  The buffer
-		name is "gdb program".
+		name is "debugged program".
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/nvim_terminal_emulator.txt:170]: I can just remove these sentences about the buffer name?

[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/doc/nvim_terminal_emulator.txt:170]: sure, i think thats fine

**Review by seandewar — APPROVED** [review]
lgtm


> also skipped all error codes above 1000 as they don't appear in our source

we do have some errors above E1000, but you already included the only relevant one in this patch in map.txt :+1:

**Comment by clason** [comment]
At some point (soon) we may have to rethink how to port runtime updates; the documentation changes are getting increasingly tedious to apply (and less and less relevant to Neovim)...

**Pull request merged**